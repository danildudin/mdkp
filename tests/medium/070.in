28 9
294 623 811 22 746 342 994 446 695 889 818 425 158 713 782 877 413 96 841 508 552 722 59 938 748 573 876 381 
136 44 10 951 265 787 46 335 555 292 876 829 822 424 852 720 329 932 619 74 88 745 103 822 352 906 300 758 8506
580 256 685 971 429 270 938 578 644 32 353 928 279 74 671 626 756 725 274 600 951 940 423 122 708 176 571 447 6106
878 90 796 4 117 97 707 218 322 782 902 80 211 696 255 979 258 175 607 595 89 507 912 347 678 86 831 650 9412
253 165 211 334 164 805 875 948 65 444 44 800 785 326 886 639 519 917 928 286 762 930 655 943 928 306 545 425 6478
607 898 485 716 901 34 298 941 306 65 395 144 23 11 199 812 288 870 772 652 920 739 377 24 27 761 140 979 9896
648 202 455 69 956 643 162 568 85 572 605 565 213 704 282 134 872 854 484 526 637 606 420 828 38 173 379 144 9552
161 102 348 533 475 803 876 609 49 959 549 859 950 622 412 913 17 291 419 894 889 161 714 291 890 742 736 913 7144
180 785 440 393 677 121 241 358 70 349 739 110 287 917 110 345 593 208 43 626 490 931 308 143 858 14 765 287 9450
713 950 633 377 554 49 749 301 927 917 936 710 523 463 692 908 429 459 383 459 48 316 59 950 208 656 154 889 5034
