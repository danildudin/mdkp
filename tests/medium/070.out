8253
11
2 4 9 13 14 18 20 21 24 25 26 
